### PR TITLE
updated TextField to AppTextField

### PR DESCRIPTION
**Pull request by nilsonsasaki**
updated TextField to an AppTextField with 25 character max input length

### CODE REVIEW

**Comment by yuumasato** [comment]
> AppTextField with 25 character max input length

Could you clarify where the input length limitatio happens?
I could not find where it is defined.

**Comment by yuumasato** [comment]
The code works, I just would like to understand where the limit is set.

**Comment by nilsonsasaki** [comment]
you can add  input formatters to textfield  "inputFormatters:[LengthLimitingTextInputFormatter(25)]," (line 125), if you use maxLenght it will show a counter of leters, 

inlobby we should add  to textfield too (line 106): 

inputFormatters: [FilteringTextInputFormatter.digitsOnly, LengthLimitingTextInputFormatter(5)],

Tested it, but didn't commit

File: lib/ui/show_news.dart [hunk 1]
```diff
@@ -116,26 +118,11 @@ class _ShowNewsScreenState extends State<ShowNewsScreen>
...
-                            hintStyle: TextStyle(color: Colors.white),
-                            focusColor: Colors.red,
-                            focusedBorder: OutlineInputBorder(
-                              borderSide: BorderSide(color: Colors.white, width: 2.0),
-                            ),
-                          ),
-                          style: const TextStyle(
-                              color: Colors.white, decoration: TextDecoration.none),
+                          labelText: 'answer'.tr,
+                          inputFormatters:[LengthLimitingTextInputFormatter(25)],
```
Comment (yuumasato) [inline lib/ui/show_news.dart:125]: Duh, totally missed this line. ;-P

**Review by yuumasato — APPROVED** [review]

**Pull request merged**